cli/mainrt.rs

```diff
@@ -70,12 +71,14 @@ fn unwrap_or_exit<T>(result: Result<T, AnyError>) -> T {
 fn main() {
   let args: Vec<_> = env::args_os().collect();
   let current_exe_path = current_exe().unwrap();
-  let standalone = standalone::extract_standalone(&current_exe_path, args);
+  let standalone =
+    standalone::extract_standalone(&current_exe_path, Cow::Owned(args));
   let future = async move {
     match standalone {
       Ok(Some(future)) => {
         let (metadata, eszip) = future.await?;
-        standalone::run(eszip, metadata).await
+        let exit_code = standalone::run(eszip, metadata).await?;
+        std::process::exit(exit_code);
       }
       Ok(None) => Ok(()),
```

Comment:
Shouldn't this error in this case instead of just exiting with exit code 0?

Reply:
I think this is correct - a few lines below we call `unwrap_or_exit(standalone_res)`

Reply:
This looks like it would be starting the process, doing nothing, then exiting with code 0 because it didn't find anything to execute. I feel like it should error instead

Reply:
I'm not sure I understand - the `Ok(None)` here says that we tried to extract the ESZip but it was not embedded in the binary. Maybe we can remove this codepath altogether now that we have `denort`. @littledivy is there a way to use `deno compile` that uses the regular binary and not the `denort` binary?

Reply:
> the `Ok(None)` here says that we tried to extract the ESZip but it was not embedded in the binary.

Yeah, exactly. Why would someone by running this code in this situation? Most likely that would indicate the trailing data got removed in which case they might be really confused that the binary just runs and exits.

Maybe I'm not understanding what this code does.